Formula/vlmcsd.rb

```diff
@@ -0,0 +1,28 @@
+class Vlmcsd < Formula
+  desc "KMS Emulator in C"
+  homepage "https://github.com/Wind4/vlmcsd"
+  url "https://github.com/Wind4/vlmcsd/archive/svn1113.tar.gz"
+  version "svn1113"
```

Comment:
I think this should be 1113, since they're all prefixed with svn

Reply:
I use `svn1113` because the author used `svn1113` on git tags

Reply:
![Screenshot_2020-05-01 Wind4 vlmcsd](https://user-images.githubusercontent.com/26341224/80765603-ba20c180-8b43-11ea-8ab9-57d3048fd50d.png)